conda/recipes/libcudf/post-link.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Only add the license notice to libcudf and not our examples / tests
+if [[ "$PKG_NAME" == "libcudf" ]]; then
+  cat ./nvlink.txt>> $PREFIX/.messages.txt
```

Comment:
nit: spaces around `>>`

Reply:
I had the same comment in my review and I pushed the fix.